pop-api/src/lib.rs

```diff
@@ -16,49 +17,16 @@ pub mod primitives;
 /// The first version of the API.
 pub mod v0;
 
-/// A result type used by the API, with the `StatusCode` as the error type.
+type ChainExtensionMethodApi = ChainExtensionMethod<(), (), (), false>;
```

Comment:
can you please add a comment for each parameter is?

Reply:
Not sure how valuable it is as most of our values are set to `()`. Thus we explain what each generic argument is but we set it to `()`, this might only cause confusion. Only in the actual api function the types are provided, but by calling functions:
```
build_read_state(BALANCE_OF)
		.input::<(TokenId, AccountId)>()
		.output::<Result<Balance>, true>()
		.handle_error_code::<StatusCode>()
		.call(&(token, owner))
```